documentation/src/docs/asciidoc/launcher-api.adoc

```diff
@@ -80,3 +80,13 @@ is currently supported via Java's `java.util.ServiceLoader` mechanism. For examp
 `junit-jupiter-engine` module registers its `org.junit.jupiter.engine.JupiterTestEngine`
 in a file named `org.junit.platform.engine.TestEngine` within the `/META-INF/services` in
 the `junit-jupiter-engine` JAR.
+
+[[launcher-api-listeners-custom]]
+==== Plugging in Your Own Test Execution Listeners
+
+In addition to the public {Launcher} API method for registering test execution listeners
+programmatically, custom {TestExecutionListener} implementations discovered at runtime via
```

Comment:
Please note that we consistently format Java types using a monospace font (e.g., `{TestExecutionListener}`). The curly brackets denote that it is a link that we have defined globally for the documentation. If a link has not been defined for the type, consider creating one or omitting the curly brackets, but do not omit the  back ticks.

Please apply this throughout your Asciidoc changes.